e2e/cli-e2e/tests/compare.e2e.test.ts

```diff
@@ -0,0 +1,58 @@
+import { simpleGit } from 'simple-git';
+import { ReportsDiff } from '@code-pushup/models';
+import { cleanTestFolder } from '@code-pushup/test-setup';
+import { executeProcess, readJsonFile } from '@code-pushup/utils';
+
+describe('CLI compare', () => {
+  const git = simpleGit();
+
+  beforeEach(async () => {
+    if (await git.diff(['--', 'examples/react-todos-app'])) {
+      throw new Error(
+        'Unstaged changes found in examples/react-todos-app, please stage or commit them to prevent E2E tests interfering',
+      );
+    }
+    await cleanTestFolder('tmp/e2e');
```

Comment:
Perhaps this should be in a subfolder - what if another E2E test has a file there that runs in parallel with this one?